src/Bicep.LangServer.IntegrationTests/HoverTests.cs

```diff
@@ -176,7 +177,8 @@ private static void ValidateHover(Hover? hover, Symbol symbol)
                     break;
 
                 case VariableSymbol variable:
-                    hover.Contents.MarkupContent.Value.Should().Contain($"var {variable.Name}: {variable.Type}");
+                    // the hovers with errors don't appear in VS code and only occur in tests
```

Comment:
>errors [](start = 39, length = 6)

This is a pre-existing issue that I've seen in other PRs as well. Not sure what the root cause is yet. It doesn't repro in VS code, however.

Reply:
I opened #1707 to track the investigation for this, but I'm considering it out of scope for this PR.